config/locales/nl.yml

```diff
@@ -1,25 +1,25 @@
 nl:
   edit: Wijzig
-  failure_when_forbidden:
-  feed_latest: RubyGems.org | Recente Gems
+  failure_when_forbidden: Controleer het webadres of probeer het opnieuw.
+  feed_latest: RubyGems.org | Nieuwste Gems
   feed_subscribed: RubyGems.org | Geabonneerde Gems
-  footer_about: RubyGems.org is de gem hosting service van de Ruby community. Publiceer and installeer je gems direct. Gebruik de API om meer informatie van beschikbare gems te vinden. Word een deelnemer en verbeter de site met jouw aanpassingen.
+  footer_about: RubyGems.org is de gem hosting service van de Ruby community. <a href="%{publish_docs}">Publiceer</a> en <a href="%{install_docs}">installeer</a> je gems direct. Gebruik <a href="%{api_docs}">de API</a> om meer informatie over beschikbare gems te vinden. <a href="%{contributing_docs}">Word een deelnemer</a> en verbeter de site met jouw aanpassingen.
   form_disable_with: Een moment geduld...
   help_url: http://help.rubygems.org
   locale_name: Nederlands
-  none:
-  not_found:
+  none: Geen
+  not_found: Niet Gevonden
```

Comment:
I believe we don't need the capital `G`

Reply:
This one also has a capital letter in the English version

Reply:
Hm, maybe that need to change also. I believe the grammar rules are that it should not have a capital in Dutch.